### ISSUE TITLE
[4.0.0] The client_authenticator_type Default value has been deleted

### ISSUE THREAD

**Issue by vincent-petit** [mrparkers/terraform-provider-keycloak#754]
I found an issue in the provider version 4.0.0 (moving from 3.10.0 to 4.0.0).
The Confidential openid client created resource are broken in admin console and on login page when the client_authenticator_type is not set in the terraform resource
```
resource "keycloak_openid_client" "my-client" {
  realm_id      = keycloak_realm.myrealm.id
  client_id     = "my-client"
  client_secret = var.my-client-secret

  access_type = "CONFIDENTIAL"
}
```

Reading the commits of the 4.0.0, i saw that the default value `"client-secret"` has been deleted.
See: 
https://github.com/mrparkers/terraform-provider-keycloak/commit/f6ab358f8f288d36c6e8af2aee71137cc435cff3#diff-95b47f453b29152c3e4fd929931b5e878b5b4e5faba33982c79229983aea35f4L74

**Comment by mrparkers** [comment]
You're right, that shouldn't have been changed.  I fixed this in #755

**Comment by mrparkers** [comment]
Thanks for reporting this issue, v4.0.1 will be released shortly.
